logic/src/commonTest/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapperTest.kt

```diff
@@ -252,6 +252,26 @@ class ProtoContentMapperTest {
         assertEquals(encryptionAlgorithm, result.encryptionAlgorithm)
     }
 
+    @Test
+    fun givenExpirableTextContent_whenMappingToProtoDataAndBack_thenTheContentsShouldMatchTheOriginal() {
+        val messageContent = MessageContent.Text("Hello")
```

Comment:
Can we also add a test at least for asset content?

Reply:
sure <3 thanks for the reminder